settings/provisioner/distro/rhel-7.2.yml

```diff
@@ -0,0 +1,8 @@
+image:
+  file: rhel-guest-image-7.2-20160302.0.x86_64.qcow2
+  server: "{{ !lookup private.images.rhel.7.2.remote_file_server }}"
+name: rhel
```

Comment:
Why do we need this data?
We are using images to specify the distro, so we don't need to hold it in yaml

```yaml
name: rhel
full_version: "7.2"
 version:
   major: '7'
   minor: '2'
```